kpops/component_handlers/kafka_connect/kafka_connect_handler.py

```diff
@@ -24,24 +25,43 @@ def __init__(self, connect_wrapper: ConnectWrapper):
         self._connect_wrapper = connect_wrapper
 
     async def create_connector(
-        self, connector_config: KafkaConnectorConfig, *, dry_run: bool
+        self,
+        connector_config: KafkaConnectorConfig,
+        *,
+        state: ConnectorState | None,
+        dry_run: bool,
     ) -> None:
         """Create a connector.
 
         If the connector exists the config of that connector gets updated.
 
         :param connector_config: The connector config.
+        :param state: The state that the connector should have afterwards.
         :param dry_run: Whether the connector creation should be run in dry run mode.
         """
         if dry_run:
             await self.__dry_run_connector_creation(connector_config)
```

Comment:
Can you add the state change to the dry-run?

Reply:
changed